evaluator/evaluate_packages.go

```diff
@@ -243,10 +244,14 @@ func vmaasResponse2UpdateDataJSON(updateData *vmaas.UpdatesV2ResponseUpdateList)
 			continue
 		}
 		// Create correct entry for each update in the list
-		pkgUpdates = append(pkgUpdates, models.PackageUpdate{
+		pkgUpdate := models.PackageUpdate{
 			EVRA:     upNevra.EVRAString(),
 			Advisory: upData.GetErratum(),
-		})
+		}
```

Comment:
ok, just please create a new sub-method to keep it clear. This method is getting bigger now.